client/src/main/java/com/arcbees/beestore/client/application/widget/ProductView.java

```diff
@@ -57,6 +60,24 @@
     }
 
     @Override
+    protected void onAttach() {
+        String loaded = $(image).parent().data("loaded");
+        if (loaded != null) {
+            return;
+        }
+
+        $(image).parent().css("display", "none");
+        $(image).parent().data("loaded", "");
```

Comment:
You don't need to execute `$().parent()` a second time. `css(..., ...).data(..., ...)` will work